src/pages/Travel/ManageTrips.tsx

```diff
@@ -24,11 +22,13 @@ const tripsFeatures: FeatureListItem[] = [
     },
 ];
 
+const ILLUSTRATION_WIDTH = 190;
+const ILLUSTRATION_HEIGHT = 172;
```

Comment:
Hm, just checking. Why height here is different from what we have in `variables.ts`?
`sectionIllustrationHeight: 220`

Reply:
Resolution is set according to [this comment](https://github.com/Expensify/App/issues/42560#issuecomment-2129612739)